Gopkg.toml

```diff
@@ -41,6 +41,10 @@
   name = "github.com/stretchr/testify"
   version = "1.1.4"
 
+[[constraint]]
```

Comment:
Sorry I did not get why is this required ? Is this for examples section ?

Reply:
Yes I had issues running the examples without pinning the version of Heimdall to a "recent" SHA. I just tried to reproduce my issue but it all works fine so we can close this PR.
Thanks!

Reply:
👍